notebooks/README.md

```diff
@@ -0,0 +1,5 @@
+## Directory structure
```

Comment:
When/why anyone should be using these notebooks??

Can you add a word in this readme about when/why to use these notebooks?